UndertaleModTool/Editors/UndertaleCodeEditor.xaml.cs

```diff
@@ -679,40 +681,43 @@ private async Task DecompileCode(UndertaleCode code, bool first, LoaderDialog ex
 
                     if (decompiled != null)
                     {
-                        string[] decompiledLines;
+                        StringReader strReader;
+                        StringBuilder strBuilder;
                         if (gettext != null && decompiled.Contains("scr_gettext"))
                         {
-                            decompiledLines = decompiled.Split('\n');
-                            for (int i = 0; i < decompiledLines.Length; i++)
+                            strReader = new(decompiled);
+                            strBuilder = new();
+                            string line;
+                            while ((line = strReader.ReadLine()) is not null)
                             {
-                                var matches = Regex.Matches(decompiledLines[i], "scr_gettext\\(\\\"(\\w*)\\\"\\)");
-                                foreach (Match match in matches)
+                                Match match = gettextRegex.Match(line);
```

Comment:
Is it possible for a line to have multiple regex matches? Aka if something like `scr_gettext\\"\w\"\ ; scr_gettext\\"\www\"\` appeared

Reply:
I didn't know that you could do that, though it's possible only with enabled "Profile mode"